From bfb965796de90829ee5e93e570eac730a18cce34 Mon Sep 17 00:00:00 2001
From: Martin Abente Lahaye <tch@sugarlabs.org>
Date: Fri, 10 Jan 2020 11:47:30 -0300
Subject: [PATCH 1/2] Fix drawing comic boxes

There is no corresponding context.restore, so I assume
this is some sort of left over from a refactoring.

Remove this context.save, otherwise it causes all sorts
of rendering glitches.
---
 globos.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/globos.py b/globos.py
index 57d473d..c47045d 100644
--- a/globos.py
+++ b/globos.py
@@ -643,7 +643,6 @@ class Grito(Globo):
                                  font_name)
 
     def imprimir(self, context):
-        context.save()
         self.draw_exclamation(context, self.x, self.y, self.ancho, self.alto,
                               self.direccion, self.punto)
         # se dibuja el correspondiente texto
-- 
2.23.0

